test/test_two_links_fixed_joint.cpp

```diff
@@ -44,9 +44,9 @@
 #include "robot_state_publisher/joint_state_listener.h"
 
 
-using namespace ros;
-using namespace tf;
-using namespace robot_state_publisher;
+// using namespace ros;
```

Comment:
Commenting out these lines will cause these tests to stop building. Please remove the comments.